### PR TITLE
Additional mass points for charged Higgs to tb analysis

### PR DESCRIPTION
**Pull request by vischia**

### CODE REVIEW

**Comment by vischia** [comment]
Hi,

I uploaded the wrong cards by mistake. I will approve the PR later today with the actual cards that I used to create the gridpacks.

Please do not approve the PR yet :)

Cheers,
Pietro

**Comment by perrozzi** [comment]
ping

**Comment by vischia** [comment]
pong

Waiting for PC greenlight, I will upload the rest of the stuff later today anyways :)

**Comment by vischia** [comment]
Hi,

here it is. You can merge now, thanks!!!

Best,
Pietro

**Comment by vischia** [comment]
Hi,

here it is. You can merge now, thanks!!!

Best,
Pietro

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M10000/ChargedHiggs_heavy_TB_M10000_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#import model 2HDMtypeII
+import model 2HDMtII_NLO
+# Define multiparticle labels
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+# Specify process(es) to run
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M10000/ChargedHiggs_heavy_TB_M10000_proc_card.dat]: can you please cleanup the datacards?

Comment (vischia) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M10000/ChargedHiggs_heavy_TB_M10000_proc_card.dat]: OK, I will do it shortly

**Comment by perrozzi** [comment]
ping

**Comment by perrozzi** [comment]
ping

**Comment by perrozzi** [comment]
closing the pr. it can be re-opened if needed

**Comment by vischia** [comment]
Hi, I just pushed the cleanup commit.

Do I open another PR or can this be re-opened?

The idea is to go for this week's MccM.

Thanks in advance!

Cheers,
Pietro

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M10000/XXX_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#
+# set seed 1
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M10000/XXX_madspin_card.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M1500/ChargedHiggs_heavy_TB_M1500_customizecards.dat [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+#set param_card mass 6 175.0
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M1500/ChargedHiggs_heavy_TB_M1500_customizecards.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M1500/ChargedHiggs_heavy_TB_M1500_proc_card.dat [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+#import model sm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M1500/ChargedHiggs_heavy_TB_M1500_proc_card.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M1500/XXX_madspin_card.dat [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#
+# set seed 1
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M1500/XXX_madspin_card.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/ChargedHiggs_heavy_TB_M2500_proc_card.dat [hunk 6]
```diff
@@ -0,0 +1,46 @@
+#************************************************************
+#*                        MadGraph 5                        *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/ChargedHiggs_heavy_TB_M2500_proc_card.dat]: cleaup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/ChargedHiggs_heavy_TB_M2500_customizecards.dat [hunk 7]
```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+#set param_card mass 6 175.0
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/ChargedHiggs_heavy_TB_M2500_customizecards.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/ChargedHiggs_heavy_TB_M2500_proc_card.dat [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+#import model sm
+#import model 2HDMtypeII
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/ChargedHiggs_heavy_TB_M2500_proc_card.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/XXX_madspin_card.dat [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#
+# set seed 1
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M2500/XXX_madspin_card.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M5000/XXX_madspin_card.dat [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#
+# set seed 1
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M5000/XXX_madspin_card.dat]: cleanup

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M7000/XXX_madspin_card.dat [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#
+# set seed 1
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ChargedHiggs_heavy_TB_M7000/XXX_madspin_card.dat]: cleanup

**Comment by perrozzi** [comment]
some more cleanup needed before to merge :-)

**Comment by vischia** [comment]
Hi,

cleanup done (also in a couple more files than indicated).

I think everything should be fine for merging the PR now!

Cheers,
Pietro

**Pull request merged**